.golangci.yml

```diff
@@ -24,6 +24,8 @@ linters-settings:
   goconst:
     min-len: 5
     min-occurrences: 4
+  lll:
+    line-length: 180
```

Comment:
Editing anything CI or style wise is only acceptable after at least some discussion. This is not only here in k6 but anywhere. 
As a whole the golangci.yml is under ... [development](https://github.com/loadimpact/k6/pull/976) as we find how golangci is working for us and how much of the things it reports are worth working with. 
We also already had a [discussion](https://github.com/loadimpact/k6/pull/943#discussion_r263732027) and agreed that the 120 default is good enough. 
As a whole if somewhere what golangci is reporting seems unreasonable and fixing it will introduce problem you can use [`//nolint` comments](https://github.com/golangci/golangci-lint#false-positives). We have already done this for `lll` in particular as there are places that it makes it more readable to just be on one line.